aws_cloudwatch_policies/aws_cloudwatch_sensitive_loggroup_encrypted.py

```diff
@@ -0,0 +1,13 @@
+from fnmatch import fnmatch
+
+# replace the log groups ARN regex with the
+# log groups that should be encrypted with your CMK
+SENSITIVE_LOG_GROUP_ARN_REGEXS = {"*LogGroup-2*"}
```

Comment:
Maybe set this to `{}` and put an example in a comment? It would short-circuit the conditional in L9. Unless this is an actual log group used by standard Panther?

Reply:
I'm guessing the example is in there for the unit test, although we might be able to mock the global variable? At least I think we can mock variables in addition to functions.

Reply:
Yea, it was for the unit tests, and I can't seem to get them to pass using mocks - should the mocks work with variables? Quite likely I'm doing something wrong

Reply:
Mocks have to be strings. If your "actual" thing is something other than a string, you need to evaluate it to turn it back into whatever it's supposed to be. I did it with [this](https://github.com/panther-labs/panther-analysis/blob/0a2524a92a335c09b9972ceaf81fded5006e3c10/standard_rules/unusual_login.py#L51), but there's probably an easier way heh.

Reply:
ahh, well - since this is already a "configuration required" policy, I'm going to vote we leave the test data in the set for now. Adding a check for a str and casting will double the size of this particular policy